### DISCUSSION TITLE
Cannot install scripts; userChromeJS working

### DISCUSSION THREAD

**Discussion by VLM-TechEd** [xiaoxiaoflood/firefox-scripts#166] [General Support]
Hi there. I am feeling frustrated. I have successfully installed userChromeJS and StyloiaX, as well as Tab Mix Plus (from onemen). But every time I try to download a script from this page, https://github.com/xiaoxiaoflood/firefox-scripts, when I right-click and select Save Target As, I end up with either a .txt file or an .htm file in my chrome folder. And nothing works. What to do? Thank you.

**Comment by TFWol** [comment]
What does the inside of the text file look like?

Like this?
```js
// ==UserScript==
// @name            BeQuiet
// @author          xiaoxiaoflood
// @include         main
// @startup         UC.beQuiet.exec(win);
// @shutdown        UC.beQuiet.destroy();
// @onlyonce
// ==/UserScript==

// inspired by https://addons.mozilla.org/en-US/firefox/addon/be-quiet/

UC.beQuiet = {
```

To reduce complications:
![image](https://user-images.githubusercontent.com/9045213/182003488-2c5ecd0d-51df-4107-b10e-3f234ad51740.png)

**Reply by VLM-TechEd** [reply]
Hi again, I'm not sure what to make of the "To reduce complications" entry. I can barely read it, and I don't know what bearing it may have on my question. Probably just my ignorance; I apologize for not "getting it".

**Comment by VLM-TechEd** [comment]
Yes, they look like that.

**Reply by 117649** [reply]
Then you can just rename it to .uc.js ......

**Comment by VLM-TechEd** [comment]
Okay, I’ll try that. They all go into the chrome folder, correct? Or do they go into a subfolder?
